Horn.fs

```diff
@@ -117,6 +116,7 @@ module Pretty =
         | AMul [ x; y ] -> printBop "*" x y
         | AMul(x :: y :: xs) -> printInt (AMul((AMul [ x; y ]) :: xs))
         | ADiv(x, y) -> printBop "/" x y
+        | AMod(x, y) -> failwith "unexpected modulo"
```

Comment:
We checked with Andrey, and HSF indeed doesn't support modulo. IDK whether the same is true for muZ3 once we get that working...